### PR TITLE
update validator to allow script[amp-story-dvh-polyfill] for transformed documents

### PR DESCRIPTION
**Pull request by erwinmombay**
This is needed to fix a visual bug when amp-story-1.0.css is inlined or loaded through link[stylesheet]

### CODE REVIEW

File: validator/validator-main.protoascii [hunk 1]
```diff
@@ -3492,6 +3492,28 @@ tags: {
   }
 }
 
+# Fixes a visual bug when amp-story-1.0.css is Server Side Rendered.
```
Comment (gmajoulet) [inline validator/validator-main.protoascii:3495]: Nit: I think we should call it "dvh polyfill"

**Review by gmajoulet — APPROVED** [review]

File: validator/validator-main.protoascii (hunk 1 above)
Comment (erwinmombay) [reply validator/validator-main.protoascii:3495]: @gmajoulet should i rename the attribute too you think?

Comment (erwinmombay) [reply validator/validator-main.protoascii]: ill go ahead and update it

**Review by jridgewell — COMMENTED** [review]
LGTM, but I'm deferring to @ampproject/wg-caching

**Comment by erwinmombay** [comment]
anyone from @ampproject/wg-caching can take a look?

**Comment by Gregable** [comment]
@amaltas is the current rotation victim

**Comment by erwinmombay** [comment]
@amaltas PTAL

**Comment by erwinmombay** [comment]
@amaltas friendly ping

**Comment by erwinmombay** [comment]
not sure who is on rotation in @ampproject/wg-caching but could anyone take a look? thank you

**Review by amaltas — APPROVED** [review]

File: validator/validator-main.protoascii [hunk 2]
```diff
@@ -3492,6 +3492,28 @@ tags: {
...
+  unique: true
+  mandatory_parent: "HEAD"
+  attrs: {
+    name: "amp-story-dvh-polyfill"
+    value: ""
+    mandatory: true
+    dispatch_key: NAME_VALUE_DISPATCH
+  }
+  cdata: {
+    # Only valid if contents are exactly either of these:
```
Comment (twifkak) [inline validator/validator-main.protoascii:3511]: Remove "either of"? I only see one option.

**Review by twifkak — COMMENTED** [review]
Hi! Minor nit I happened to notice.

File: validator/validator-main.protoascii (hunk 2 above)
Comment (erwinmombay) [reply validator/validator-main.protoascii:3511]: will follow up! ty

**Pull request merged**